hadoop-crypto/src/main/java/com/palantir/crypto2/hadoop/DelegatingFileSystem.java

```diff
@@ -81,4 +84,9 @@ public void copyToLocalFile(boolean delSrc, Path src, Path dst, boolean useRawLo
                 : getLocal(conf);
         FileUtil.copy(this, src, local, dst, delSrc, conf);
     }
+
+    @Override
+    public BlockLocation[] getFileBlockLocations(Path path, long start, long len) throws IOException {
```

Comment:
So did we lose https://github.com/palantir/hadoop-crypto/pull/92 when moving to `FilterFileSystem` in https://github.com/palantir/hadoop-crypto/pull/94 ?